plugins/services/src/js/components/modals/ServiceRootGroupModal.tsx

```diff
@@ -0,0 +1,361 @@
+                  misc: (
+                    <Trans>You do not have permission to create a group.</Trans>
+                  )
+                },
+                isPending: false
+              });
+              return;
+            default:
+              this.setState({
+                errors: {
+                  misc: <span>Unable to create group: {e.message}</span>
+                },
+                isPending: false
+              });
+              return;
+          }
+        }
+      });
+  }
+
+  getSaveAction() {
+    const createContext = {
+      data: this.state.data
+    };
+
+    return dl.query(groupCreateMutation, createContext);
+  }
+
+  getHeader() {
+    return (
+      <FullScreenModalHeader>
+        <FullScreenModalHeaderActions
+          actions={[
+            {
+              className: "button-primary-link button-flush-horizontal",
+              clickHandler: this.handleClose,
+              label: "Cancel"
+            }
+          ]}
+          type="secondary"
+        />
+        <FullScreenModalHeaderTitle>
+          <Trans>New Group</Trans>
+        </FullScreenModalHeaderTitle>
+        <FullScreenModalHeaderActions
+          actions={[
+            {
+              className: "button-primary flush-vertical",
+              clickHandler: this.handleSave,
+              label: "Save"
+            }
+          ]}
+          type="primary"
+        />
+      </FullScreenModalHeader>
+    );
+  }
+
+  getModalContent() {
+    const { data, isPending, errors } = this.state;
+    return (
+      <form className="container" onChange={this.handleFormChange}>
+        {this.getMiscErrors()}
+        <Trans render="h1" className="flush-top short-bottom">
+          General
+        </Trans>
+        <FormRow>
+          <FormGroup
+            className="column-12 column-medium-4"
+            showError={Boolean(errors.name)}
+          >
+            <FieldLabel>
+              <FormGroupHeading required={true}>
+                <Trans render={<FormGroupHeadingContent primary={true} />}>
+                  Name
+                </Trans>
+              </FormGroupHeading>
+            </FieldLabel>
+            <FieldAutofocus>
+              <FieldInput
+                name="name"
+                type="text"
+                value={data.name}
+                disabled={this.props.isEdit || isPending}
+              />
+            </FieldAutofocus>
+            <FieldError>{errors.name}</FieldError>
+          </FormGroup>
+        </FormRow>
+        <FormRow>
+          <FormGroup className="column-12">
+            <FieldLabel>
+              <FormGroupHeading>
+                <Trans render={<FormGroupHeadingContent primary={false} />}>
+                  Path
+                </Trans>
+              </FormGroupHeading>
+            </FieldLabel>
+            <div>{`/${data.name}`}</div>
+          </FormGroup>
+        </FormRow>
+      </form>
+    );
+  }
+
+  getMiscErrors() {
```

Comment:
Is there a reason we couldn't use the `ErrorsAlert` component directly?

Reply:
nope, I just didn't even think about using `ErrorsAlert` I can swap that out.

Reply:
So `ErrorsAlert` is specific to errors from a service and expect the errors to be in a specific format, and parses them through ErrorUtils. I wanted similar markup but didn't want to be confined to the error data structure used for the create service form.

Reply:
What are misc errors supposed to be?